Wire-iOS/Sources/Initializer.swift

```diff
@@ -0,0 +1,100 @@
+//
+// Wire
+// Copyright (C) 2020 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+import WireSyncEngine
+import WireCommonComponents
+import avs
+
+// MARK: - Initializer
+public protocol Initializer {
+    func configure()
+}
```

Comment:
The name `Initializer` is misleading because you're not necessarily constructing or instantiating anything. The name `configure` is also confusing because... what are you configuring?

Naming suggestion:

```suggestion
public protocol LaunchSequenceOperation {
    func execute()
}
```

Reply:
Of course, It is more understandable your suggestion.